trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,159 @@
+# Trace Context HTTP Header Format
+
+This document describes the binding of the [trace context](README.md) to `traceparent`
+and `tracestate` http headers.
+
+## Relationship between the headers
+
+The `tracestate` header is not an extension of the data in `traceparent`,
+rather it is the gold copy per vendor. For example, the below hints that there
+is only one trace graph and that is from the Congo service:
+
+```
+trace-parent: 00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01
+trace-state: congo=BleGNlZWRzIHRo
+ZSBzaG9ydCB2ZWhlbWVuY2Ugb2YgYW55IGNhcm5hbCBwbGVhc3VyZS4=
+```
+
+There is one exception to the "gold copy" rule, which is when the incoming
```

Comment:
I agree with @adriancole on moving this generic case section elsewhere. I think there is some good potential value in the option, but I'm not sure the benefit outweighs the cost at the moment. The costs as I see it:

* It adds to the cognitive load required to understand the spec, and is not required for successful execution of an impl (i.e. if nobody does this then the spec still works).
* In order for this to work, an impl needs to recognize when the incoming generic case is not compatible with their tracing system, and when they see that they have to do something special or else the vendor using the generic case's trace graph is busted. The "something special" is copying the `traceparent` value into the generic vendor's state. e.g. in this case my tracing system would have to set `tracestate: mystuff=someOpaqueValue,yelp=00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01` (possible base64-ing it if we decide that's required for the opaque vendor payload values). If I fail to do that copy, then the next time a yelp system sees this trace context they will not be able to resume correctly - their trace graph will be broken and probably unfixable. 
* The above issue is tricky to explain and potentially easy to goof up. It increases the complexity of an impl and goes against the general ethos we seemed to agree on in the workshop that you shouldn't mess with another vendor's gold copy. And when someone does goof up it means breaking other vendors' trace graphs.

For these reasons I'd recommend tabling this feature/option for now, and focus on the bare minimum to allow the spec to reach its goals. The less moving parts and the simpler it is, the higher likelihood of success IMO. Like I said I do think there's value here and it's worth revisiting, I just don't think it needs to be here right now. Unless we think it needs to be now or never, in which case it's probably worth discussing further.

Reply:
done